### DISCUSSION TITLE
Resize buttons

### DISCUSSION THREAD

**Discussion by MICKNICK911** [TomSchimansky/CustomTkinter#387] [General]
How do I resize buttons as root window gets bigger or smaller?
Will appreciate an example.

Keep up with the good work

**Comment by TomSchimansky** [comment]
You need to use the grid geometry manager and assign a weight to the rows and columns with `app.grid_rowconfigure(<row>, weight=<weight>)` and a weight greater than 0.
Have a look at this example: https://github.com/TomSchimansky/CustomTkinter/wiki/App-structure-and-layout#small-onefile-example-app-with-grid
